src/components/Form.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+import plus from "../assets/plus.svg";
+import { taskCreated } from "../features/todos";
+import { useDispatch } from "react-redux";
+
+export default function Form() {
+  const [text, setText] = useState("");
+  const dispatch = useDispatch()
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    if (text.trim()) {
+        dispatch(taskCreated(text))
+        setText("");
+    }
+  }
+  const handleChange = (event) => {
+    const { value } = event.target;
+    setText(value);
+  }
```

Comment:
You don't need to create this function because it is only used once. what I can suggest is to use the arrow function inside the attribute of your event. like this:
```suggestion
 onChange={(e) => setText(e.target.value)}}
```